python/lsst/ip/diffim/imageDecorrelation.py

```diff
@@ -282,13 +298,24 @@ def run(self, scienceExposure, templateExposure, subtractedExposure, psfMatching
         # The subtracted exposure variance plane is already correlated, we cannot propagate
         # it through another convolution; instead we need to use the uncorrelated originals
         # The whitening should scale it to expVar + matchedVar on average
-        varImg = correctedExposure.variance.array
-        # Allow for numpy type casting
-        varImg[...] = preSum*preSum*exposure.variance.array + kSumSq*matchedExposure.variance.array
+        if self.config.completeVarPlanePropagation:
+            self.log.debug("Complete variance plane calculation")
```

Comment:
I think these log messages need to be raised to `info`, and re-worded to be more meaningful in the logs.